bedrock/careers/templates/careers/diversity.html

```diff
@@ -26,65 +26,80 @@ <h1>Diversity & inclusion at Mozilla</h1>
     </div>
   </header>
 
-  <!-- Diversity intro -->
   <section class="c-careers-intro">
     <div class="mzp-l-content mzp-t-content-md">
-      <h2>Belong where you work</h2>
+      <h2 class="c-careers-section-title">
```

Comment:
**Suggestion:** should we also change this background color into the lighter teal to match the other headers?

Reply:
D'oh, this was lost with the rest of the listings PR